sonic_y_cable/y_cable.py

```diff
@@ -856,20 +865,28 @@ def get_ber_info(physical_port, target):
             physical_port).write_eeprom(curr_offset, 1, buffer)
         if result is False:
             return result
+        time_start = time.time()
         while(True):
             done = platform_chassis.get_sfp(physical_port).read_eeprom(curr_offset, 1)
-            y_cable_validate_read_data(done, 1, physical_port, "BER data ready to read")
+            if y_cable_validate_read_data(done, 1, physical_port, "BER data ready to read") == EEPROM_READ_DATA_INVALID:
+                return EEPROM_ERROR
+            time_now = time.time()
+            time_diff = time_now - time_start
             if done[0] == 1:
                 break
+            elif time_diff >= WAIT_TIMEOUT_FOR_BER_EYE:
+                return EEPROM_TIMEOUT_ERROR
 
         idx = 0
-        maxLane = 2
+        maxLane = 4
```

Comment:
Suggest making this a constant and defining at top of file, since it is used in multiple places.

Reply:
fixed